packaging/os/apt.py

```diff
@@ -246,11 +246,9 @@ def package_status(m, pkgname, version, cache, state):
         else:
             return False, False, False
     try:
-        has_files = len(pkg.installed_files) > 0
-    except UnicodeDecodeError:
-        has_files = True
+        has_files = ll_pkg.current_state == apt_pkg.CURSTATE_CONFIG_FILES
```

Comment:
should this not be a or? does current_State == apt_pkg.CURSTATE_CONFIG_FILES when there are installed files?